ckanext/push_errors/tests/test_error_logger.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests for the log_error function in error_logger.py.
```

Comment:
Drop this file from #20
We just need tests for `get_error_trace_id` and `get_error_log_id` and the see how to continue